src/resources/packages/classy/functions/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './dateUtils';
+export * from './utils';
```

Comment:
Rename the file to `url` and export it as such. Let's avoid generic `utils` where possible.